cmake/pickFortranCompilerFlags.cmake

```diff
@@ -8,16 +8,26 @@ if ( NOT Fortran_FLAGS_INIT )
     "Enable compiler run-time checks? (Enabling this will turn off most compiler optimizations.)" )
   mark_as_advanced ( ENABLE_RUNTIME_CHECKS )
 
+  if ( "${CMAKE_SYSTEM_NAME}" STREQUAL "Windows" )
+    set(pre /)
+    set(spc :)
+    set(Q /Q)
+  else()
+    set(pre -)
+    set(spc " ")
+    set(Q -)
+  endif()
```

Comment:
Intel flags are (oh, so helpfully) different on Windows. This is a small abstraction to allow compilation on Windows without ifort trying to interpret *nix flags as files.

Reply:
This is a good trick! I might steal this for something else! :)